src/backend/App.js

```diff
@@ -1,10 +1,10 @@
 const express = require('express')
```

Comment:
Could you please add semicolons here?  Yeah I know it  comes from the starting guide of Express,  but still  feels a bit odd to me!